### PR TITLE
doc: add minutes for meeting 17 July 2024

### PR DESCRIPTION
**Pull request by mhdawson**

### CODE REVIEW

File: meetings/2024-07-17.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+    conversation along.
+    * Benjamin, ok to land and iterate, but we allow as much breakage etc.
+    * Matteo, 2 takes.
+      * First one is that we should try to have meeting with the Typescript team. Should wait until
+        we get their feedback. Seems like the asked for some time. Ok with landing and removing
+        later
+    * Marco
+      * Currently blocked as swc having trouble with some architectures so still some time before
+         it would land anyway.
+      * Feedback from TypeScript team is very favorable. Created package to wrap swc which
```
Comment (marco-ippolito) [inline meetings/2024-07-17.md:74]:
```suggestion
      * Feedback from TypeScript team is that there are some concerns that can be solved. Created package to wrap swc which
```

**Comment by mhdawson** [comment]
@nschonni, @Trott I see you two had the most involvement in setting up the linting. It's been driving me a bit crazy that it complains about whitespace but does not identify what line is the problem.  Is that the way it should be working?

**Comment by nschonni** [comment]
it seems like the https://github.com/nodejs/TSC/blob/main/.github/workflows/remark-lint-problem-matcher.json isn't commenting on the particular line, because the output isn't including the line details anymore. I'll see if I can figure out what broke.

**Comment by nschonni** [comment]
OK, did some digging, and it looks like that rule is actually a custom one from @Trott https://github.com/Trott/remark-lint-no-trailing-spaces. Maybe it needs an update for an api change in remark.

EX: I updated the `remark-preset-lint-node` to the latest, and it flagged a few new things (and correctly indicated the line/column of the issue)

```
$ npm test

> tsc@1.0.0 test
> remark -qf .

meetings\io.js\2015-04-08.md
16:52-16:72 warning Unexpected reference to undefined definition, expected corresponding definition (`comment by @mikeal`) for a link or escaped opening bracket (`\[`) for regular text                       no-undefined-references remark-lint
17:50-17:92 warning Unexpected reference to undefined definition, expected corresponding definition (`comments by @silverwind and @fishrock123`) for a link or escaped opening bracket (`\[`) for regular text no-undefined-references remark-lint

ONBOARDING.md
            warning Remove trailing whitespace                                                                   
                                                                                              no-trailing-spaces      remark-lint

⚠ 3 warnings
```

File: meetings/2024-07-17.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+  * Antoine, only the compile time option is on the table because of the architectures side of
+    Things. Marco believe latest version from swc will be addressed
+  * Geoffrey, land but hold back from release, but increases the chances of backports
+  * Richard, likely not a Node.js 22.x release in next few weeks anyway.
+  * Michael
+    * one approach, land in main with compile time option, backport that to 22.x, then remove
+      the compile time option in main. That keeps most code consistent for backporting while
+      not enabling in 22.x in shipping versions. If its not ready for 23 we could then re-add
+      the compile time guard (or not if it is ready)
+
```
Comment (nschonni) [inline meetings/2024-07-17.md:130]:
```suggestion
```

**Comment by mhdawson** [comment]
@nschonni thanks for looking at the linter.

**Comment by Trott** [comment]
@nschonni @mhdawson I think when https://github.com/nodejs/remark-preset-lint-node/pull/574 lands, that will generate a new release of the lint preset that will fix the issue with reporting lines and columns for trailing spaces. Unfortunately, that's currently blocked by the wonkiness we're seeing with Node.js 22 that I'm sure everyone is aware of and eager to see fixed.

**Comment by Trott** [comment]
OK, @mhdawson! If you rebase against origin/main and run `npm ci`, you should be all set. The trailing spaces linter will now tell you where to look.

**Comment by mhdawson** [comment]
@nodejs/tsc could be good to get some approvals so I can land.

@Trott since the linter is green on this one, I'll look forward to the info for future PRs. Thanks for your help in getting it fixed.

**Review by marco-ippolito — APPROVED** [review]

**Review by RafaelGSS — APPROVED** [review]

File: meetings/2024-07-17.md [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+## Agenda
+
+### Announcements
+
+* Rafael about to promote 22.5.0 !
+
+### Reminders
+
+* Remember to nominate people for the [contributor spotlight](https://github.com/nodejs/node/blob/main/doc/contributing/reconizing-contributors.md#bi-monthly-contributor-spotlight)
```
Comment (richardlau) [inline meetings/2024-07-17.md:31]: Link has now moved: https://github.com/nodejs/node/pull/53990
```suggestion
* Remember to nominate people for the [contributor spotlight](https://github.com/nodejs/node/blob/main/doc/contributing/recognizing-contributors.md#bi-monthly-contributor-spotlight)
```

**Review by richardlau — APPROVED** [review]

**Pull request merged**